src/app/page.tsx

```diff
@@ -1,7 +1,100 @@
+import Link from 'next/link';
+import { LinkButton } from '../components/LinkButton';
+import style from './page.module.scss';
+
 export default function Home() {
   return (
-    <main>
-      <h1>Home</h1>
-    </main>
+    <div className={style.container}>
+      <main className={style.main}>
+        <div className={style.hero}>
+          <img src='/images/hero.png' alt='Hero' className={style.heroImage} />
+          <div className={style.breadcrumb}>
+            <Link href='/' className={style.breadcrumbLink}>
+              Top
+            </Link>
+          </div>
+        </div>
+        <div className={style.contents}>
+          <h1 className={style.contentTitle}>活動内容</h1>
+          <p className={style.aboutText}>
+            Maximumでは
+            <span className={style.span}>
+              <span className={style.bold}>競技プログラミング</span>
+            </span>
+            (週2回)と
+            <span className={style.span}>
+              <span className={style.bold}>Web研究会</span>
+            </span>
+            (週1回)の2つの活動を行っています。
+          </p>
+          <div className={style.imgBox1}>
+            {/* <img src='/images/hero.png' alt='Hero' className={style.img} /> */}
+          </div>
+          <div className={style.activityCP}>
+            <div className={style.activityCPtitle}>
+              <h2>競技プログラミング</h2>
+              <p>Competitive programming</p>
+            </div>
+            <p className={style.activityCPcontent}>
+              <span className={style.span}>
+                <span className={style.bold}>AtCoder</span>
+              </span>
+              &nbsp;や&nbsp;
+              <span className={style.span}>
+                <span className={style.bold}>ICPC</span>
+              </span>
```

Comment:
clsx が入ってるので `import clsx from 'clsx'` したら

```suggestion
              <span className={clsx(style.span, style.bold)}>ICPC</span>
```

で OK

Reply:
ほかのところも